### ISSUE TITLE
Websocket bufReader and bufWriter are outdated

### ISSUE THREAD

**Issue by WhoAdarshPandya** [zhmushan/abc#106]
Problem : 
i've been using latest deno version **1.3.0** , i tried to work with **Websocket(ws) standard library** , it complains that type **bufReader** and **bufWriter** in **https://deno.land/std/ws/mod.ts (version 0.65.0)** is not compatible with version **https://deno.land/std/ws@0.63.0/mod.ts (version 0.63.0)** , however i changed version of **ws to v 0.63.0** and it's working, but it's not working with latest version

versions:
**deno : v1.3.0
abc : 1.0.2
ws: std(version 0.65.0) [works fine with 0.63.0]**

**Comment by zhmushan** [comment]
We can fix broken deps in third-party modules through import map
```json
{
  "imports": {
    "https://deno.land/std@0.63.0/ws/mod.ts":"https://deno.land/std@0.65.0/ws/mod.ts"
  }
}
```

Of course, I have released 1.0.3 https://github.com/zhmushan/abc/pull/108, you can directly use the new version to get the fix

**Comment by WhoAdarshPandya** [comment]
Thanks
